ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -72,7 +72,7 @@ - (NSDictionary *)constantsToExport {
 - (id)init
 {
     if ((self = [super init])) {
-#if !TARGET_OS_TV
+#if (!TARGET_OS_TV && !TARGET_OS_OSX)
```

Comment:
I recognize it's draft and you're just getting compile to work, but seems OSX should likely have battery information as well? Laptop use case?

Reply:
Yup, still need to implement that.

Documented here: https://developer.apple.com/documentation/kernel/iopmpowersource#//apple_ref/doc/uid/TP40012438